spec/Classes.tex

```diff
@@ -888,10 +888,10 @@ \section{The {\em this} Method}
 \index{this@\chpl{this}}
 \index{classes!this@\chpl{this}}
 
-A procedure method declared with the name \chpl{this} allows a class to be
-``indexed'' similarly to how an array is indexed.  Indexing into a
-class instance has the semantics of calling a method
-named \chpl{this}.  There is no other way to call a method
+Declaring a procedure method with the name \chpl{this} allows the
+class's objects to be ``called'' or ``indexed,'' similarly to how
```

Comment:
I suggest making it clearer by saying:

... to be ``called'' similarly to how a function is called or ``indexed''
similarly how an array is indexed.